### PR TITLE
DOCSP-31387 source database ref page

### PR DESCRIPTION
**Pull request by jocelyn-mendez1**
Source database ref page.

STAGING: https://docs-mongodbcom-staging.corp.mongodb.com/docs-relational-migrator/docsworker-xlarge/DOCSP-31387/supported-databases/ 

JIRA: https://jira.mongodb.org/browse/DOCSP-31387

BUILD: https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=64bac0d687b3b36046cbb9a3

### CODE REVIEW

File: source/supported-databases.txt [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+Definition
+----------
+
+To create a Relational Migrator project, you can connect to a source database to migrate
+to MongoDB. This page provides the supported source databases. 
+
+.. note::
+
+   It may be possible to use other versions or deployments but they have not been tested
```
Comment (ajhuh-mdb) [inline source/supported-databases.txt:25]: [suggestion][ Active voice ](https://www.mongodb.com/docs/meta/style-guide/writing/use-active-voice/)(making user the subject) 

```suggestion
   You might be able to use other versions or deployments for your source database, but they have not been tested 
```

File: source/supported-databases.txt [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+Relational Migrator supports migration from source databases to MongoDB. 
+
+Definition
+----------
+
+To create a Relational Migrator project, you can connect to a source database to migrate
+to MongoDB. This page provides the supported source databases. 
+
+.. note::
```
Comment (ajhuh-mdb) [inline source/supported-databases.txt:23]: [non-blocking suggestion] I feel like this could be a warning admonition instead of a note. Thoughts?

**Review by ajhuh-mdb — APPROVED** [review]
LGTM % a couple suggestions!

File: source/supported-databases.txt (hunk 1 above)
Comment (jocelyn-mendez1) [reply source/supported-databases.txt]: Done.

File: source/supported-databases.txt (hunk 2 above)
Comment (jocelyn-mendez1) [reply source/supported-databases.txt]: Yes, thank you!

**Comment by jocelyn-mendez1** [comment]
Hi @tomhollander, I have added the the supported source databases and versions. Let me know of any possible changes. Thanks!

**Review by tomhollander — CHANGES_REQUESTED** [review]
Content looks good. However I don't think it's placed properly in the ToC.
This doesn't really belong under "Migration scenarios and use cases", and with this placement it seems impossible to read the parent topic. I think this would be better off as a sibling to that topic, i.e. directly under the "Relational Migrator" root?

**Review by tomhollander — APPROVED** [review]

**Pull request merged**